integration/env/helper.go

```diff
@@ -0,0 +1,17 @@
+package env
+
+import (
+	"fmt"
+	"os"
+)
+
+func getEnv(name string) string {
+	return os.Getenv(name)
+}
+
+func mustGetEnv(name string) string {
```

Comment:
Since we refactor this and make it nicer we can return an error to the caller and only panic in a couple of exclusive cases at the very core of the program initialization. This would then maybe better called something like `getEnvOrError`. Or better, simply `getEnv` and return a specific error type when the value is empty, which we can ignore if we want to, or deal with if we should.

Reply:
I thought about this as well. In that case would get like 5 extra lines for each env var and we loose immediate visibility which of them are optional. And what can really we do if the required value isn't initalized? Yes we can exit with 1 and print error message which is bit more elegant but for me it isn't enough to pay the price of code explosion.

Reply:
But in general it makes sense. Otherwise `BuildAWS`, `BuildAzure`, `BuildKVM` won't really work as they will panic because of variables missing for other provider. I'll change that.